pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/TableOp.java

```diff
@@ -92,4 +101,40 @@ boolean runOp() {
     }
     return true;
   }
+
+  private boolean createSchema() {
+    try {
+      ControllerTest.sendPostRequest(
+          ControllerRequestURLBuilder.baseUrl(CONTROLLER_URL).forSchemaCreate(),
+          FileUtils.readFileToString(new File(_schemaFileName)));
+      return true;
+    } catch (IOException e) {
+      System.out.println(e.getMessage());
```

Comment:
Let us change this to an error log for now. We can then configure logs appropriately to get whatever we want on the console.
Also, include the  table config file path on all log messages